apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html

```diff
@@ -160,6 +161,69 @@ <h1>{{log.operationType}}
 
         <div class="address-row-wrapper d-flex">
             <div class="address-row mt-05">
+                <!-- account address block start -->
+                <ng-container *ngIf="log && log?.details && log?.details?.addresses && log?.details?.addresses.length">
+                    <ng-container *ngFor="let item of log?.details?.addresses; let index = index">
+                        <div class="single-address-row" *ngIf="index===0">
```

Comment:
HTML repetition can be avoided with 
`<div class="single-address-row" *ngIf="index===0 || index>0 && showSingleAddress && isShowMultipleDataIndex===parentIndex">`